presto-cassandra/src/main/java/com/facebook/presto/cassandra/NativeCassandraSession.java

```diff
@@ -101,6 +101,17 @@ public NativeCassandraSession(String connectorId, JsonCodec<List<ExtraColumnMeta
     }
 
     @Override
+    public String getCassandraVersion()
+    {
+        ResultSet result = executeWithSession(session -> session.execute("select release_version from system.local where key = 'local'"));
```

Comment:
We can remove this where condition, return same result irrespective of where condition.